charts/velero/templates/_helpers.tpl

```diff
@@ -122,3 +100,10 @@ For examples:
 {{- $minorVersion := .Capabilities.KubeVersion.Minor | regexFind "[0-9]+" -}}
 {{- printf "%s.%s" .Capabilities.KubeVersion.Major $minorVersion -}}
 {{- end -}}
+
+{{/*
+Create the BackupStorageLocation accessMode
+*/}}
+{{- define "velero.backupStorageLocation.accessMode" -}}
+{{ default "ReadWrite" .accessMode }}
+{{- end -}}
```

Comment:
I think we could remove it here, adding the default access mode `ReadWrite` in the `template/backupstoragelocation.yaml`.